build/ci/informix/setup-informix.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+printf "\nCopy database connection resource file"
+mkdir --parents --verbose ~/.geotools
+cp --verbose --force ./build/ci/informix/informix-sqli.properties ~/.geotools/
+
+printf "\nCreate GEOTOOLS schema...\n"
+docker cp ./build/ci/informix/setup_informix.commands geotools:/home/informix/
+docker exec -u informix -w /opt/ibm/informix/ -i geotools /opt/ibm/informix/bin/dbaccess sysmaster /home/informix/setup_informix.commands
```

Comment:
I need some help with this step, as it doesn't work right now.
any ideas @GeorgeDewar ?

Reply:
according to this: https://www.ibm.com/docs/en/informix-servers/14.10?topic=access-example-run-command-file the syntax is valid

Reply:
Hi Mark, thanks for taking this on. If it helps, I used the **ibmcom/informix-innovator-c** image, rather than ibmcom/informix-developer-database. I can't remember why though so that info is of limited use...

The commands I used manually to set this container up to work with the tests were:

```
create database geotest with log;
database geotest;
create table test1(
    shape st_point
);
drop table test1;
```

Creating a table is necessary to cause the spatial blade (i.e. extension) to initialize.

None of this is directly answering why your command isn't working, though - I will have a look at that now.

Reply:
I've found the problem, I think. The command you are running inside the container is:

```
dbaccess sysmaster /home/informix/setup_informix.commands
```

But Informix expects the file to have a .sql extension, and for the .sql extension to be omitted in the name passed to dbaccess. So, it appears that it works if you rename setup_informix.commands to setup_informix.sql, and then invoke dbaccess like `dbaccess sysmaster /home/informix/setup_informix`.

Reply:
still getting the same error message when the command executes: `-23101 Unable to load locale categories.` even after adding the `-e DB_LOCALE=en_US.819 -e CLIENT_LOCALE=en_US.819` options; I think I'll need to fiddle with the PATH

I get the same message when running the command from a shell in the container, but after exporting both those variable in the shell and running the command things work out.
```sh
[informix@15477085410d ~]$ export CLIENT_LOCALE=en_US.819
[informix@15477085410d ~]$ export DB_LOCALE=en_US.819
[informix@15477085410d ~]$ dbaccess sysmaster /home/informix/setup_informix

Database selected.

Collation set.

Database closed.

Database created.

Database closed.
```

so that should be fixable